common/reportMetrics.py

```diff
@@ -369,108 +373,343 @@ def process_replay(file_path):
+                                to_send_metric_data = []
+                                current_line_count = 0
+                                chunk_count += 1
+                            for i in range(0, len(row)):
+                                if field_names[i] == "timestamp":
+                                    current_row[field_names[i]] = row[i]
+                                    if min_timestamp_epoch == 0 or min_timestamp_epoch > long(row[i]):
+                                        min_timestamp_epoch = long(row[i])
+                                    if max_timestamp_epoch == 0 or max_timestamp_epoch < long(row[i]):
+                                        max_timestamp_epoch = long(row[i])
+                                else:
+                                    column_name = field_names[i].strip()
+                                    metric_key = column_name.split("[")[0]
+                                    if column_name.find("]") == -1:
+                                        column_name = column_name + "[-]"
+                                    # Generate normalization id or use from config.ini
+                                    if column_name.find(":") == -1:
+                                        group_id = get_normalization(grouping_map, metric_key)
+                                        column_name = column_name + ":" + str(group_id)
+                                    elif len(column_name.split(":")[1]) == 0:
+                                        group_id = get_normalization(grouping_map, metric_key)
+                                        column_name = column_name + str(group_id)
+                                    elif len(column_name.split(":")[1]) != 0:
+                                        if len(normalization_ids_map) != 0:
+                                            if metric_key in normalization_ids_map:
+                                                group_id = int(normalization_ids_map[metric_key])
+                                        else:
+                                            group_id = column_name.split(":")[1]
+                                        column_name = column_name.split(":")[0] + ":" + str(group_id)
+
+                                    current_row[column_name] = row[i]
+                            to_send_metric_data.append(current_row)
+                            current_line_count += 1
+                    # send final chunk
+                    if len(to_send_metric_data) != 0:
+                        send_data([to_send_metric_data, min_timestamp_epoch, max_timestamp_epoch], file_path, chunk_count + 1)
+                        chunk_count += 1
+                    logger.debug("Total chunks created: " + str(chunk_count))
                 save_grouping(grouping_map)
 
 
+def replay_sar(metric_file_path, grouping_map):
+    print('Replaying sar file')
+    with open(metric_file_path) as metric_file:
```

Comment:
check if file exists

Reply:
added check when input as parameter, already checked at top of tag that calls this